src-vue/src/views/DeveloperView.vue

```diff
@@ -112,6 +116,33 @@ export default defineComponent({
                         position: 'bottom-right'
                     });
                 });
+        },
+        async getInstalledMods() {
+            let game_install = {
+                game_path: this.$store.state.game_path,
+                install_type: this.$store.state.install_type
+            } as GameInstall;
+            await invoke("get_installed_mods_caller", { gameInstall: game_install }).then((message) => {
+                // Simply console logging for now
```

Comment:
So this will get removed in next PR, right?

Reply:
Not the next one directly but once the feature is stabilised and release, then yes.